src/main/java/com/mabl/integration/jenkins/test/output/Properties.java

```diff
@@ -1,28 +1,45 @@
 package com.mabl.integration.jenkins.test.output;
 
+import com.fasterxml.jackson.databind.annotation.JsonAppend;
 import com.google.common.collect.ImmutableCollection;
 
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
+import java.util.ArrayList;
+import java.util.Collection;
 
 @XmlRootElement(name = "properties")
 @XmlAccessorType(XmlAccessType.FIELD)
 public class Properties {
 
     @XmlElement(name = "property")
-    private ImmutableCollection<Property> properties;
+    private Collection<Property> properties;
```

Comment:
Minor; if ordering is important, then we will want to be more specific than `Collection`.

Reply:
Make sense. Thank you